quic/s2n-quic/Cargo.toml

```diff
@@ -91,3 +91,11 @@ s2n-quic-transport = { path = "../s2n-quic-transport", features = ["unstable_res
 tokio = { version = "1", features = ["full"] }
 tracing = { version = "0.1" }
 tracing-subscriber = { version = "0.3", features = ["env-filter"] }
+
+# Compile with the `s2n_docsrs` feature to enable feature tagging in rust docs. See
+# documentation for additional configuration options: https://docs.rs/about/metadata
+[package.metadata.docs.rs]
+rustdoc-args = ["--cfg", "s2n_docsrs"]
```

Comment:
Is there a reason we need our own? Why not just use the one docs.rs already sets?

https://github.com/tokio-rs/tokio/blob/dcae2b9eb8f32eb92ecb3efe126b91577c5096e4/tokio/src/lib.rs#L637

Reply:
Yea I wasn't sure if that was something that docsrs set or if tokio didnt namespace the cfg flag. Lemme try and find docs mentioning that and/or test it on my test crate

Reply:
So actually are you certain that gets set by docsrs? Tokio also sets [rustdoc-args](https://github.com/tokio-rs/tokio/blob/dcae2b9eb8f32eb92ecb3efe126b91577c5096e4/tokio-util/Cargo.toml#L64) which specifies `docsrs`

Reply:
Testing shows that `docsrs` and that we dont need `package.metadata.docs.rs` either.
- [docs](https://docs.rs/toidiu/0.5.0/toidiu/index.html)
- [commit](https://github.com/toidiu/toidiu-rs/commit/8891a7cbc9f3319b17dcf504435bb19031bd215c)